theano/tensor/basic.py

```diff
@@ -4573,7 +4573,7 @@ def reshape(x, newshape, ndim=None, name=None):
                 "to know what the number of dimensions of the reshaped "
                 "variable will be. You can provide the 'ndim' keyword "
                 "argument to 'reshape' to avoid this problem." % newshape)
-    op = Reshape(ndim, name)
+    op = Reshape(ndim)
```

Comment:
@nouiz Do I have to specify anything in the command for the test to be run on GPU? thanks.